opentelekomcloud/services/rds/resource_opentelekomcloud_rds_instance_v3.go

```diff
@@ -1154,9 +1154,18 @@ func validateRDSv3Flavor(argName string) schema.CustomizeDiffFunc {
 		}
 		dataStoreInfo := d.Get("db").([]interface{})[0].(map[string]interface{})
 		flavor := d.Get(argName).(string)
-
+		dversion := dataStoreInfo["version"].(string)
+		// TODO: need to be removed when https://jira.tsi-dev.otc-service.com/browse/BM-1169 done
+		version, err := strconv.Atoi(dversion)
+		if err != nil {
+			return fmt.Errorf("error during version conversion: %s", dversion)
+		}
+		if version > 13 {
```

Comment:
Maybe change it to `if version == 13`?
Otherwise greater versions will be passing and from the looks of the initial bug we have problem only with `14` version.